lib/checks/visibility/hidden-content.js

```diff
@@ -1,13 +1,16 @@
-let styles = window.getComputedStyle(node);
+const whitelist = ['SCRIPT', 'HEAD', 'TITLE', 'NOSCRIPT', 'STYLE', 'TEMPLATE'];
+if (
```

Comment:
There are currently only 5 files that use this coding style - lets change them all back